src/client/blaze.ts

```diff
@@ -54,11 +59,59 @@ export const BlazeKeystoreClient = (keystore: Keystore | undefined, wsOptions: B
     };
   };
 
+  const sendMsg = (recipientID: string, category: string, data: any) => {
+    if (typeof data === 'object') data = JSON.stringify(data);
+
+    const message = {
+      id: uuid(),
+      action: 'CREATE_MESSAGE',
+      params: {
+        category,
+        recipient_id: recipientID,
+        conversation_id: uniqueConversationID(keystore!.user_id, recipientID),
+        message_id: uuid(),
+        data: base64RawURLEncode(Buffer.from(data)),
+      },
+    };
```

Comment:
We should condition `keystore`, `ws`, if undefined or null, we should throw a error.